### PR TITLE
now you only need to create font once.

### PR DESCRIPTION
**Pull request by lishaoxia1985**
if you want, you can let the font size changed with resolution and don't need create extra font.

### CODE REVIEW

**Comment by yairm210** [comment]
And how does this look? What's the performance like?

**Comment by lishaoxia1985** [comment]
first, get a font whose size is 48 (you can set it to 72 if you want), then enlarge or small the font, we can get more and more different font. it shows very good even high resolution screen and save the memory.

**Comment by lishaoxia1985** [comment]
A. you can see this web:
https://civilization.fandom.com/wiki/List_of_technologies_in_Civ5
B. of course you can write a fun() to do this, in this fun() we can maybe use some smooth method.
 you can simplify this as you wish, now i only give a method, the largest size of the font used in game is 30, so now we set the font size to 45, in the high resolution(such as 2560*1600) it shows very good, the mac is different with pc and mobile phone, i don't have these device for test. you can test it in low resolution.
i think if this method is good we can set different resolution with different font(warning: some button may not change size with the font, we should check this).

File: android/assets/jsons/Translations/NewGame,SaveGame,LoadGame,Options.json [hunk 1]
```diff
@@ -134,9 +134,9 @@
 
 	"HIGHLY EXPERIMENTAL - YOU HAVE BEEN WARNED!": {
 		Italian:"IN FASE DI PRODUZIONE - TI ABBIAMO AVVERTITO!"
-		Simplified_Chinese:"⚠︎警告！试验性功能！"
+		Simplified_Chinese:"⚠警告！试验性功能！"
 		Ukrainian:"ПЕРЕБУВАЄ В РОЗРОБЦІ — ВАС ПОПЕРЕДЖЕНО!"
-		Traditional_Chinese:"⚠︎警告！實驗性功能！"
+		Traditional_Chinese:"⚠警告！實驗性功能！"
```
Comment (lishaoxia1985) [inline android/assets/jsons/Translations/NewGame,SaveGame,LoadGame,Options.json]: unicode changed, now it will be good!

**Comment by yairm210** [comment]
I'll merhe this to a separate beach for checking when I have the time :)

**Pull request merged**